### PR TITLE
Resolve Issues

### PR DESCRIPTION
**Pull request by AlenaKhanis**
There are two irrelevant commits there about deleting db, so you can ignore them - I wrote a comment next to them.

### CODE REVIEW

File: backend/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Flask==3.0.2
+python-dotenv==1.0.1
+Flask-Cors==4.0.0
+Flask-JWT-Extended==4.6.0
+psycopg2-binary==2.9.9
+bcrypt==3.2.0
```
Comment (ofersadan85) [inline backend/requirements.txt:6]: 🙂 Nice

File: backend/db/schema.sql [hunk 2]
```diff
@@ -15,31 +15,12 @@ CREATE TABLE IF NOT EXISTS files(
     id INTEGER PRIMARY KEY NOT NULL,
     filename TEXT NOT NULL,
     filetype TEXT NOT NULL CHECK (filetype IN ('movie', 'song', 'episode', 'picture')),
-    filesize INTEGER NOT NULL,
-    created TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
-    expired TIMESTAMP,
-    description TEXT,
+    fileSize INTEGER NOT NULL,
```
Comment (ofersadan85) [inline backend/db/schema.sql:18]: See the differences here between your file version and my updated one and fix

**Review by ofersadan85 — CHANGES_REQUESTED** [review]

File: backend/db/schema.sql (hunk 2 above)
Comment (AlenaKhanis) [reply backend/db/schema.sql]: Added new commit with the fix version

**Review by ofersadan85 — APPROVED** [review]
Looks really great now!

**Review by ofersadan85 — APPROVED** [review]
Really nice work

**Pull request merged**